actioncable/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Create notion of an `ActionCable::SubscriptionAdapter`.
+    Separate out Redis functionality into `AC::SA::Redis`,
```

Comment:
`ActionCable::`SubscriptionAdapter::Redis`. We don't use abbreviations in our documentation